.github/workflows/deploy.yml

```diff
@@ -52,4 +52,8 @@ jobs:
           environment: ${{ inputs.environment }}
 
       - name: Deploy release
-        run: make release-deploy APP_NAME=${{ inputs.app_name }} ENVIRONMENT=${{ inputs.environment }}
+        run: make release-deploy \
+          APP_NAME=${{ inputs.app_name }} \
+          ENVIRONMENT=${{ inputs.environment }} \
+          DEPLOY_GITHUB_REF=${{ github.ref_name }} \
```

Comment:
@chouinar I'm pretty sure its the raw tag name, so the `2024.11.27-1` in https://github.com/HHS/simpler-grants-gov/tree/2024.11.27-1

https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables

<img width="765" alt="image" src="https://github.com/user-attachments/assets/bdbc207a-a855-4c34-a08b-870cd19ce9c0">

Reply:
Will there be a consistent URL I can build if it can refer to both a branch and a release? Any pattern of `github.com/something/<branch or release>` that I can follow?

Reply:
Err, I suppose the `/tree` path does, but for releases I wonder if linking to the release page would be better - could maybe regex that

Reply:
For both the ref and commit the URL is gonna be `https://github.com/HHS/simpler-grants-gov/tree/< commit / branch / release >`, examples:

- https://github.com/HHS/simpler-grants-gov/tree/main
- https://github.com/HHS/simpler-grants-gov/tree/44b954e85c4ca7e3714f9f988a919fae40ec3c98
- https://github.com/HHS/simpler-grants-gov/tree/2024.11.27-1